Python/pythonrun.c

```diff
@@ -65,6 +65,7 @@ static PyObject *run_pyc_file(FILE *, const char *, PyObject *, PyObject *,
                               PyCompilerFlags *);
 static void err_input(perrdetail *);
 static void err_free(perrdetail *);
+static int PyRun_InteractiveOneObjectEx(FILE *, PyObject *, PyCompilerFlags *);
```

Comment:
I'm not sure that we need to make this new function public. I propose to make it private.

Please add also a short comment to explain that it's similar to PyRun_InteractiveOneObject() but don't print the error on failure.

Reply:
I do not understand your comment, this new function is static.

Reply:
Oh, I was wrong. I understood that the function was public and exported. It's ok :-)